zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/RequestMapper.java

```diff
@@ -228,6 +227,15 @@ private static <R> int getIntOrZero(final R request, final Function<R, Integer>
     return value == null ? 0 : value;
   }
 
+  private static Map<String, Number> buildJobChangeset(final JobChangeset jobChangeset) {
+    final Map<String, Number> changeset = new HashMap<>();
+    Optional.ofNullable(jobChangeset.getRetries())
+        .ifPresent(retries -> changeset.put(JobRecord.RETRIES, retries));
```

Comment:
🔧 I'm wondering if we could use something like this Optional handling in the processor as well instead of the current routine 🤔